### PR TITLE
PAE - 658 - Replace CircleCI with GitHub Actions.

### PR DESCRIPTION
**Pull request by ivanvgh**
### Description:

This PR replaces CircleCI with GitHubActions.

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        uses: actions/checkout@v2
+
+      - name: Cache dependency # caching dependency will make our build faster.
+        uses: actions/cache@v2 # for more info checkout pip section documentation at https://github.com/actions/cache
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements/test.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: Set up Python ${{ matrix.python-version }}
```
Comment (Squirrel18) [inline .github/workflows/push.yml:26]: Please, add a blank line before this line. @ivanvgh

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

File: .github/workflows/push.yml (hunk 1 above)
Comment (ivanvgh) [reply .github/workflows/push.yml:27]: Done.

**Review by Squirrel18 — APPROVED** [review]
Thank you for taking ownership of these changes. @ivanvgh

**Review by diegomillan — APPROVED** [review]
LGTM @ivanvgh good work setting this up!

**Pull request merged**